src/Oscoin/Crypto/Address/Serialisation.hs

```diff
@@ -0,0 +1,249 @@
+{-# LANGUAGE UndecidableInstances #-}
+{-# OPTIONS_GHC -fno-warn-orphans  #-}
+module Oscoin.Crypto.Address.Serialisation
+    ( DeserialiseError(..)
+    , serialiseAddress
+    , deserialiseAddress
```

Comment:
I'm not sure this is a real concern, and I partly blame the `serialise` library for the uk spelling, but the rest of our codebase uses american english, (grep `initialize`, `unauthorized`, etc.) - so might be easier to remember than having to think about which functions have uk spelling.

Reply:
If you decide to change this, also grep for `deserialise`

Reply:
I think consistency is important 😉 I am myself into the uk spelling camp (coming from a uk-heavy working environment legacy, hehe), so I need some nudge to keep me honest on that 😁

Reply:
84c5a8f